share/shwap/sample_test.go

```diff
@@ -25,10 +25,18 @@ func TestSampleValidate(t *testing.T) {
 	for _, proofType := range []rsmt2d.Axis{rsmt2d.Row, rsmt2d.Col} {
 		for rowIdx := 0; rowIdx < odsSize*2; rowIdx++ {
 			for colIdx := 0; colIdx < odsSize*2; colIdx++ {
-				sample, err := inMem.SampleForProofAxis(rowIdx, colIdx, proofType)
+				idx, err := shwap.SampleIndexFromCoordinates(rowIdx, colIdx, odsSize)
```

Comment:
It should be `odsSize*2` which is equals to `edsSize`. which is what we get when we call `acc.Size(ctx)`

Reply:
This should also fix this Error, NoError fix

Reply:
Fixed, thanks.